qa/rpc-tests/zcpour.py

```diff
@@ -13,91 +13,38 @@
 
 class PourTxTest(BitcoinTestFramework):
     def setup_network(self):
-        # Start with split network:
-        return super(PourTxTest, self).setup_network(True)
+        self.nodes = []
+        self.is_network_split = False
+        self.nodes.append(start_node(0, self.options.tmpdir))
 
-    def send_pours_around(self):
-        zckeypair = self.nodes[1].zcrawkeygen()
+    def run_test(self):
+        zckeypair = self.nodes[0].zcrawkeygen()
         zcsecretkey = zckeypair["zcsecretkey"]
         zcaddress = zckeypair["zcaddress"]
 
-        (total_in, inputs) = gather_inputs(self.nodes[1], 40)
-        protect_tx = self.nodes[1].createrawtransaction(inputs, {})
-        pour_result = self.nodes[1].zcrawpour(protect_tx, {}, {zcaddress:49.9}, 40, 0.1)
+        (total_in, inputs) = gather_inputs(self.nodes[0], 40)
+        protect_tx = self.nodes[0].createrawtransaction(inputs, {})
+        pour_result = self.nodes[0].zcrawpour(protect_tx, {}, {zcaddress:39.9}, 40, 0.1)
 
-        receive_result = self.nodes[1].zcrawreceive(zcsecretkey, pour_result["encryptedbucket1"])
+        receive_result = self.nodes[0].zcrawreceive(zcsecretkey, pour_result["encryptedbucket1"])
         assert_equal(receive_result["exists"], False)
 
-        protect_tx = self.nodes[1].signrawtransaction(pour_result["rawtxn"])
-        self.nodes[1].sendrawtransaction(protect_tx["hex"])
-        self.nodes[1].generate(1)
+        protect_tx = self.nodes[0].signrawtransaction(pour_result["rawtxn"])
+        self.nodes[0].sendrawtransaction(protect_tx["hex"])
+        self.nodes[0].generate(1)
 
-        receive_result = self.nodes[1].zcrawreceive(zcsecretkey, pour_result["encryptedbucket1"])
+        receive_result = self.nodes[0].zcrawreceive(zcsecretkey, pour_result["encryptedbucket1"])
         assert_equal(receive_result["exists"], True)
 
-        pour_tx = self.nodes[1].createrawtransaction([], {})
-        pour_result = self.nodes[1].zcrawpour(pour_tx, {receive_result["bucket"] : zcsecretkey}, {zcaddress: 49.8}, 0, 0.1)
-
-        self.nodes[1].sendrawtransaction(pour_result["rawtxn"])
-        self.nodes[1].generate(1)
+        pour_tx = self.nodes[0].createrawtransaction([], {})
+        pour_result = self.nodes[0].zcrawpour(pour_tx, {receive_result["bucket"] : zcsecretkey}, {zcaddress: 39.8}, 0, 0.1)
 
-        print "Syncing blocks..."
-        connect_nodes(self.nodes[1], 0)
-        sync_blocks(self.nodes[0:2])
```

Comment:
There's no longer a network for nodes to sync with; it was pointless for this test.